internal/driver/metadata.go

```diff
@@ -198,3 +213,43 @@ func getImageFormats(d *usbdevice.Device) (interface{}, error) {
 	}
 	return r, nil
 }
+
+func getSupportedIntervalFormats(d *usbdevice.Device) (interface{}, error) {
+	descs, err := d.GetFormatDescriptions()
+	if err != nil {
+		return nil, err
+	}
+	type result struct {
+		IntervalFormats [][][]v4l2.FrameIntervalEnum
+	}
```

Comment:
I am confused on the result format and why the triple nested array? Shouldn't it be more of an array of objects (which have sub arrays)? This way its hard to tell what the distinction between separate arrays means.